eng/pipelines/azure-pipelines.yml

```diff
@@ -99,7 +99,7 @@ stages:
 
       - job: windows
         # timeout accounts for wait times for helix agents up to 30mins
-        timeoutInMinutes: 60
+        timeoutInMinutes: 90
```

Comment:
Is signing adding 30mins?

Reply:
Sometimes, yes unfortunately. It really depends on how backed up the signing service is, but we have seen it add at least 20-30 minutes. Some of the files that we sign are also pretty big, for instance, the MSIs for the workloads which contain dcp binaries, those take a long time to upload, wait for signing service to process, and then download back for signing validation. Now that we have Helix runs, it is likely that we'll need more than 30 extra minutes, but I guess we'll find out soon.